scripts/release_notes.tpl.txt

```diff
@@ -0,0 +1,91 @@
+Please check out [CHANGELOG](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.5.md) for a full list of changes. And make sure to read [upgrade guide](https://github.com/etcd-io/website/blob/main/content/en/docs/v3.6/upgrades/upgrade_3_6.md) before upgrading etcd (there may be breaking changes).
```

Comment:
Note; we wanted to simplify the release notes. https://github.com/etcd-io/etcd/issues/15185

Reply:
I agree with that issue/improvement, but I'd like to avoid scope creeping this pull request and implementing it by only creating the release with `gh`. I think it would be a good idea to create an umbrella issue for the improvements we want to make to the release process. I'll note this and create it when I close some of the tasks I currently have open.

Reply:
We can do it separately. This PR only focuses on automating the creating github release.

Should we create separate templates for each release (including 3.5.x, 3.4.x and 3.6.x)? @ivanvc Since this PR is for the main branch, we can resolve it in followup PR if you want.

Reply:
Ah, good point. I didn't realize that it has mixed links from 3.5 and 3.6 in the release notes. I'll work on it in this PR.